user_docs/en/userGuide.t2t

```diff
@@ -592,6 +592,12 @@ Moving to a list item's containing object will take you back to the list.
 You can then move past the list if you wish to access other objects.
 Similarly, a toolbar contains controls, so you must move inside the toolbar to access the controls in the toolbar.
 
+If you yet prefer to move back and forth between every single object on the system, you can use the move to previous/next object in a flattened view commands, respectively.

```

Comment:
I think this will read slightly better

```suggestion
If you yet prefer to move back and forth between every single object on the system, you can use commands to move to the previous/next object in a flattened view.
```